threedi_modelchecker/tests/test_checks_other.py

```diff
@@ -293,6 +294,69 @@ def test_cross_section_location(session):
     assert len(errors) == 1
 
 
+@pytest.mark.parametrize(
+    "shape, width, height, ok",
+    [
+        # --- closed cross-sections ---
+        # shapes 0, 2, 3 and 8 are always closed
+        *[
+            (
+                i,
+                "0 4.142 5.143 5.143 5.869 5.869",
+                "0 0.174 0.348 0.522 0.696 0.87",
+                False,
+            )
+            for i in [0, 2, 3, 8]
```

Comment:
Types 0, 2, 3, 8 are indeed always closed, but as there is string parsing involved anyway, it is good to have the "width" and "height" right.

0, 1 should have a scalar value as "width" and "height"
2, 3, 8 should have a scalar value as "width" and no value (NULL) as "height"
(circle doesn't even have

Reply:
In overstroming_vanuit_bergermeer all the entries of shape 1 except some dummy values have a NULL height. Is this incorrect?